content/en/opentelemetry/schema_semantics/semantic_mapping.md

```diff
@@ -146,3 +218,9 @@ Enabling this option adds both the OpenTelemetry resource attributes and the Dat
 [2]: /getting_started/tagging/unified_service_tagging#opentelemetry
 [3]: https://opentelemetry.io/docs/specs/semconv/resource/container/
 [4]: https://github.com/open-telemetry/semantic-conventions/releases/tag/v1.27.0
+[5]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/attributesprocessor
+[6]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/transformprocessor
+[7]: https://opentelemetry.io/docs/concepts/signals/traces/#server
+[8]: https://opentelemetry.io/docs/concepts/signals/traces/#client
+[9]: https://opentelemetry.io/docs/specs/semconv/attributes-registry/db/#db-system
+[10]: #otel-database-system-types---datadog-span-type
```

Comment:
```suggestion
[10]: #mapping-opentelemetry-database-system-type-to-datadog-span-type
```
Please verify this against the actual link - I'm not certain mine will match exactly.